src/main/java/frc/robot/subsystems/DriveBase.java

```diff
@@ -35,7 +36,16 @@
     private DriveMode driveMode;
 
     // The drivebase talons
-    private WPI_TalonSRX leftFrontTalon, leftRearTalon, rightFrontTalon, rightRearTalon, talon5plyboy;
+    private static WPI_TalonSRX leftFrontTalon, leftRearTalon, rightFrontTalon, rightRearTalon, talon5plyboy;
```

Comment:
Since Drivebase is a singleton we should use a pattern that does not require static variables or static methods. e.g. always using `DriveBase.getInstance()`.

Reply:
OK, fixed that. Committed new code, awaiting feedback.